packages/app/src/app/pages/Sandbox/Editor/Workspace/Dependencies/VersionEntry/index.js

```diff
@@ -77,8 +77,8 @@ export default class VersionEntry extends React.PureComponent {
       '00383ecd8441ead30b1b0ff981c426f5'
     );
     const index = client.initIndex('npm-search');
-    index.search({ query: dependency, hitsPerPage: 1 }, (err, { hits }) => {
-      const versions = Object.keys(hits[0].versions).sort((a, b) => {
+    index.getObject(dependency, ['versions']).then(({ versions: hits }) => {
+      const versions = Object.keys(hits).sort((a, b) => {
```

Comment:
I'd keep calling this versions though, since hits is a list of packages usually

Reply:
oh I see the problem now, the variable was declared twice

Reply:
The names clash, that's why I renamed it.

Im gonna rename to results, is that a better name?

Reply:
```
index.getObject(dependency, ['versions']).then(result =>
  const versions = Object.keys(result.versions).sort((a, b) => {
```

This could work too